spec/features/edit_document_associations_spec.rb

```diff
@@ -36,22 +44,28 @@ def and_i_click_on_edit_associations
 
   def then_i_can_see_the_current_selections
     @request = stub_publishing_api_put_content(Document.last.content_id, {})
-    expect(page).to have_select("associations[association_id][]",
+    expect(page).to have_select("associations[multi_association_id][]",
                                  selected: "Initial association")
+    expect(page).to have_select("associations[single_association_id][]",
+                                selected: "Selected association")
   end
 
   def when_i_edit_the_associations
-    select "Association to select 1", from: "associations[association_id][]"
-    select "Association to select 2", from: "associations[association_id][]"
-    unselect "Initial association", from: "associations[association_id][]"
+    select "Association to select 1", from: "associations[multi_association_id][]"
+    select "Association to select 2", from: "associations[multi_association_id][]"
+    unselect "Initial association", from: "associations[multi_association_id][]"
 
+    select "Other association", from: "associations[single_association_id][]"
     click_on "Save"
   end
 
   def then_i_can_view_the_associations
     expect(page).to have_content "Association to select 1"
     expect(page).to have_content "Association to select 2"
     expect(page).not_to have_content "Initial association"
+
+    expect(page).to have_content "Other association"
+    expect(page).not_to have_content "Selected association"
```

Comment:
Could we use a ‘within’ block here (or more specific text) to make it clearer which select box we’re testing?

Reply:
Unfortunately there isn't a good selector to use for a within - had a look at adding one but there isn't really an obvious attribute to use.